examples/TwoD_MultipleBodies.jl

```diff
@@ -0,0 +1,29 @@
+using WaterLily
+using StaticArrays
+using CUDA
```

Comment:
`Bodies` is currently not working with CUDA, plus it is not used in this example anyways.
```suggestion
```